build.sbt

```diff
@@ -0,0 +1,28 @@
+val appName = "address-lookup-performance-tests"
+val appVersion = "0.1.0-SNAPSHOT"
+
+lazy val root = (project in file("."))
+  .enablePlugins(GatlingPlugin)
+  .enablePlugins(CorePlugin)
+  .enablePlugins(JvmPlugin)
+  .enablePlugins(IvyPlugin)
+  .enablePlugins(SbtAutoBuildPlugin)
+  .settings(
+    organization := "uk.gov.hmrc",
+    name := appName,
+    version := appVersion,
+    scalaVersion := "2.12.12",
+    scalacOptions ++= Seq("-feature"),
```

Comment:
What is the base set of scalac options that this is adding to?  The gatling docs currently suggest:

```
scalacOptions := Seq(
  "-encoding", "UTF-8", "-target:jvm-1.8", "-deprecation",
  "-feature", "-unchecked", "-language:implicitConversions", "-language:postfixOps")
```

https://gatling.io/open-source/

Reply:
Previously I have also seen the following in HMRC repos:

`"-Xlint", "-Xmax-classfile-name", "100"`

Reply:
The base set of Scalac options comes now from `sbt-auto-build` plugin https://github.com/hmrc/sbt-auto-build/blob/master/src/main/scala/uk/gov/hmrc/SbtAutoBuildPlugin.scala#L35.
Which in turn retrieves the details from: https://github.com/hmrc/sbt-settings/blob/master/src/main/scala/uk/gov/hmrc/DefaultBuildSettings.scala#L27

 Even though this plugin has been in `plugins.sbt` it is not enabled by default. So this has been enabled now.